src/utils/mtev_log.h

```diff
@@ -172,6 +172,11 @@ API_EXPORT(int)
   mtevL(ls, "[FATAL] " args); \
   abort(); \
 } while(0)
+#define mtevAssert(condition) do {\
+  if(!(condition)) { \
+    mtevFatal(mtev_error, "%s\n", #condition);\
+  }\
+} while(0)
```

Comment:
also, the mtevFatal string should be something like `mtevAssert(mtev_error, "assertion (%s) at %s:%d failed\n", #condition, __FILE__, __LINE__);`